.konflux/operator/konflux.Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23@sha256:4805e1cb2d1bd9d3c5de5d6986056bbda94ca7b01642f721d83d26579d333c60 as builder
+
+WORKDIR /go/src/github.com/openshift-kni/numaresources-operator
+COPY . .
+
+ENV GOEXPERIMENT=strictfipsruntime
+ENV CGO_ENABLED=1
+ENV GOTAGS="strictfipsruntime"
+
+# Build
+RUN make binary-all
+
+FROM registry.redhat.io/rhel9-4-els/rhel-minimal:9.4@sha256:9577a9ed1707ba2a1a229559d188a015cf3b20b18e4b83541f427697d1c0b8df
```

Comment:
as previuosly stated, updating the builders digests will be extremely painful as it would be purely manual, I would suggest to remove the digests to enable the automation to pick up the latest digest until mintmaker decide to support sub-directory bumps

Reply:
dockerfile digests will be updated in subdirs (rpm lockfiles will not be as you stated)

Reply:
That may depend on renovate configuration but yes this is working in the other repos